### ISSUE TITLE
`eslint-plugin-lit-a11y` issue in `role-supports-aria-attr` rule

### ISSUE THREAD

**Issue by akornmeier** [open-wc/open-wc#2466]
## Expected behavior

Execute ESLint a11y rules against code with known issues and receive proper results.

## Actual Behavior

The `role-supports-aria-attr` rule appears to be missing a valid role type check that is causing rule logic to run on an undefined variable. Details below:

```
TypeError: Cannot destructure property 'props' of 'roles.get(...)' as it is undefined.
Occurred while linting /lit-element-starter-ts/src/my-element.ts:41
Rule: "lit-a11y/role-supports-aria-attr"
    at Object.enterElement (/lit-element-starter-ts/node_modules/eslint-plugin-lit-a11y/lib/rules/role-supports-aria-attr.js:46:32)
    at visit (/lit-element-starter-ts/node_modules/eslint-plugin-lit/lib/template-analyzer.js:257:29)
    at /lit-element-starter-ts/node_modules/eslint-plugin-lit/lib/template-analyzer.js:264:25
    at Array.forEach (<anonymous>)
    at visit (/lit-element-starter-ts/node_modules/eslint-plugin-lit/lib/template-analyzer.js:263:30)
    at TemplateAnalyzer.traverse (/lit-element-starter-ts/node_modules/eslint-plugin-lit/lib/template-analyzer.js:272:9)
    at TaggedTemplateExpression (/lit-element-starter-ts/node_modules/eslint-plugin-lit-a11y/lib/rules/role-supports-aria-attr.js:39:20)
    at ruleErrorHandler (/lit-element-starter-ts/node_modules/eslint/lib/linter/linter.js:1114:28)
    at /lit-element-starter-ts/node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
```

**Comment by akornmeier** [comment]
Issue fixed (https://github.com/open-wc/open-wc/pull/2472).
